docs/conf.py

```diff
@@ -1,188 +1,69 @@
-# -*- coding: utf-8 -*-
+# Configuration file for the Sphinx documentation builder.
 #
-# Core ML documentation build configuration file
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-import coremltools
-import sys
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
 import os
-import re
+import sys
+sys.path.insert(0, os.path.abspath('.'))
 
-for m in [
-    "converters",
-    "utils",
-]:
-    module_name = "coremltools." + m
-    sys.modules[module_name] = eval(module_name)
 
-sys.path.insert(0, os.path.abspath("."))
+# -- Project information -----------------------------------------------------
 
-# -- General configuration ------------------------------------------------
+project = 'coremltools API Reference'
+copyright = '2021, Apple Inc'
+author = 'Apple Inc'
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
+# The full version, including alpha/beta/rc tags
+release = '4.1'
```

Comment:
it's going to be brittle if we need to remember to update this line each time we release a new version. It should getting this from the installed coremltools. After `import coremltools`, you should be able to change this line to
`release = coremltools.__version__`.

Reply:
Got an error with `release = coremltools.__version__`, obviously coremltools was not imported first. Since this release option is not needed, I commented it out. I haven't made any changes to the build-docs.sh script and I'm not proposing any at this time, but that is probably where we would add a coremltools import.